project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=0.13.16-M1
```

Comment:
@dwijnand What are the stability guarantees of 0.13.16-M1? Why was that version not published as 0.13.16?

When can we expect 0.13.16 final? Apparently we're blocking the entire Scala eco-system from even trying out sbt 1.0.0-RC2, so ...